parts/k8s/kubernetesmastercustomdata.yml

```diff
@@ -372,9 +372,7 @@ runcmd:
 # the first arg is the number of retries, the second arg is the wait duration between two retries and the rest of the args are the cmd to run
 - set -x
 - . /opt/azure/containers/provision_source.sh
-- retrycmd_if_failure 120 1 5 nc -zuw1 $(grep nameserver /etc/resolv.conf | cut -d \  -f 2) 53
```

Comment:
remove "check for network availability before proceeding" steps because they aren't meaningful